lib/features/mailbox/presentation/mailbox_view.dart

```diff
@@ -31,7 +31,7 @@ class MailboxView extends GetWidget<MailboxController> {
             left: false,
             child: RefreshIndicator(
               color: AppColor.primaryColor,
-              onRefresh: () async => controller.refreshGetAllMailboxAction(),
+              onRefresh: () async => controller.refreshAllMailbox(),
```

Comment:
why not call `controller.refreshMailboxChanges`?

Reply:
refreshMailboxChanges is only called when there is a change in one of the mailboxes. Here the user wants to refresh all. If the first time they do not get the mailbox back, they can get it all back from the server or local saved.